docs/src/examples/contingent-portfolio-programming.md

```diff
@@ -232,9 +240,7 @@ application_value = @expression(model, [i=1:n_DP, j=1:n_CT, k=1:n_DA, l=1:n_CM],
 
 ```julia
 optimizer = optimizer_with_attributes(
-    () -> Gurobi.Optimizer(Gurobi.Env()),
-    "IntFeasTol"      => 1e-9,
-    "LazyConstraints" => 1,
```

Comment:
I'll have to look into HiGHS and lazy constraints. These formulations don't really need lazy constraints, though...